pinot-tools/src/main/java/org/apache/pinot/tools/UpsertQuickStart.java

```diff
@@ -48,13 +48,12 @@ public static void main(String[] args)
   public void execute()
       throws Exception {
     File quickstartTmpDir = new File(FileUtils.getTempDirectory(), String.valueOf(System.currentTimeMillis()));
-    File configDir = new File(quickstartTmpDir, "configs");
-    File dataDir = new File(quickstartTmpDir, "data");
-    Preconditions.checkState(configDir.mkdirs());
+    File bootstrapTableDir = new File(quickstartTmpDir, "meetupRsvp");
+    File dataDir = new File(bootstrapTableDir, "data");
     Preconditions.checkState(dataDir.mkdirs());
 
-    File schemaFile = new File(configDir, "upsert_meetupRsvp_schema.json");
-    File tableConfigFile = new File(configDir, "upsert_meetupRsvp_realtime_table_config.json");
+    File schemaFile = new File(bootstrapTableDir, "meetupRsvp_schema.json");
```

Comment:
this change is not right?

Reply:
this the schema name that we copied from the resource package.

Reply:
Right. But any reason to rename it?

Reply:
to match the directory convention to load the table:
the schema and table config file names are in the format below.
<table_name>/<table_name>_schema.json
<table_name>/<table_name>_offline_table_config.json